src/tailwind/plugin-options.js

```diff
@@ -17,6 +17,9 @@ module.exports = {
         paddingTop: theme('spacing.24'),
         paddingBottom: theme('spacing.24')
       }
+    },
+    '.history-query.suggestion-group:hover': {
```

Comment:
This removes the hover decorations to all HQ, including the ones in the empathize.
You can deactivate the text decoration in the history query with tailwind.

Reply:
This is still here and it's not needed